pkg/object/trafficcontroller/trafficcontroller.go

```diff
@@ -99,12 +97,6 @@ var _ easemonitor.Metricer = (*TrafficObjectStatus)(nil)
 
 func init() {
 	supervisor.Register(&TrafficController{})
-	api.RegisterObject(&api.APIResource{
```

Comment:
Why delete this registration?

Reply:
cycle import, i move this registration to api pkg. it can't be solved by other way... i try it several times.